shop/src/main/java/ro/msg/learning/shop/configuration/OrderStrategyConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package ro.msg.learning.shop.configuration;
+
```

Comment:
Try to get used to having a separator of maximum 1 empty line. There's never necessary to have 2 or more

Reply:
removed redundant lines